<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>160</width>
      <height>84</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTrigg">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelTrigg">
       <property name="text">
        <string>Trigger</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="spinBoxCycleNum">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Number of cycles</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxW">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelW">
       <property name="text">
        <string>Noise wight</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>184</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelImport">
       <property name="text">
        <string>Import file name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditImport">
       <property name="text">
        <string>WH.dat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFreq">
       <property name="text">
        <string>Frequency column name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFreq">
       <property name="text">
        <string>Fgen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPhase">
       <property name="text">
        <string>Phase column name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPhase">
       <property name="text">
        <string>Theta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelExportFileName">
       <property name="text">
        <string>Export File Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditExport">
       <property name="text">
        <string>FitedData.txt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>461</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
